src/python/espressomd/thermostat.pyx

```diff
@@ -391,39 +382,29 @@ cdef class Thermostat(object):
             mpi_bcast_parameter(FIELD_TEMPERATURE)
             mpi_bcast_parameter(FIELD_NPTISO_G0)
             mpi_bcast_parameter(FIELD_NPTISO_GV)
-        
-            
-    IF DPD or INTER_DPD:
-        @AssertThermostatType(THERMO_DPD, THERMO_INTER_DPD)
-        def set_dpd(self, **kwargs):
+
+    IF DPD:
+        @AssertThermostatType(THERMO_DPD)
```

Comment:
We should allow multiple thermostats in the case of DPD

Reply:
If DPD is know to work with, say Langevin, you can simply modify this line to
```python
@AssertThermostatType(THERMO_DPD,THERMO_LANGEVIN)
```
Please do not remove `@AssertThermostatType` altogether, as I'm pretty sure that DPD and LB do *not* play together nicely.

Reply:
I'm not sure why there would be problems for any combination, what would be the problem with LB and DPD? Also we've used such combinations in production without issues.

Reply:
So we want to remove that line?

Reply:
I think DPD should work with all the other thermostats.

Reply:
That means removing the line is appropriate